### PR TITLE
Markdown style for Exercism

### PR DESCRIPTION
**Pull request by kotp**
This addresses:

* single sentence per line
* minor formatting changes
* minor tyopgraphical fixes

Message-Id: <20211224204253.1630934-3-keeperotphones@gmail.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -83,12 +71,11 @@ We use the following terminology:
 
 ### Language Track Repositories
 
-We've given each language track an ID, which is a url-friendly version of the
-language name. For example, _C++_ has the ID `cpp`. This ID is used throughout
-the Exercism ecosystem.
+We have given each language track an ID, which is an url-friendly version of the language name.
```
Comment (wolf99) [inline CONTRIBUTING.md:74]: `an url-` vs `a URL-`?
The latter seems more correct to me. But it's not a big deal.

**Review by wolf99 — APPROVED** [review]

**Comment by kotp** [comment]
On 21/12/25 01:17AM, wolf99 wrote:
> > @@ -83,12 +71,11 @@ We use the following terminology:
>  
> `an url-` vs `a URL-`?
> The latter seems more correct to me. But it's not a big deal.

I believe that the use of "an" is based on the following sound.  So if
"universal" is as a consonant "y" sound, then yes, I believe it should
be "a universal resource…" rather than "an universal resource…".  But if
it is pronounced as a vowel, then I think it is correct to say "an".
Even with a literal consonant, we say "an", such as "an hour" rather
than "a hour", and write it the same way, again, because of it being
based on the vowel or consonant sound.

Please do let me know if in English that is a "y" consonant sound for
"universal".  Write me an "h", write me an "i", write me a "u".

**Comment by wolf99** [comment]
I have the same understanding of the grammatical guide for a vs an... My point isabout how people prefer to use "url" and the resulting choice of a or an.

URL is an acronym, usually spelled out when read out, like "you are ell". Meaning that "a" should be used. 

If we consider it instead to be a word in its own right, "url" (which I think is what you are following @kotp), then it should be "an".

I guess I have no strong feeling as to which is chosen, moreso pointing out that it _could_ be seen as more correct with the acronym way if we are pedantic about acronyms vs word for URL/url 😄.

**Comment by kotp** [comment]
On 21/12/25 01:57PM, wolf99 wrote:
> If we consider it instead to be a word in its own right, "url" (which
> I think is what you are following @kotp), then it should be "an".

No, I am not thinking that you would say "oorl", for example, but I also
do not have experience in how people say it out loud.  I assume they
read acronyms as what they stand for.  Such as reading lol, as "laugh
out loud" as I have never heard anyone actually say "lol", or "rotf",
rather than "rolling on the floor".

I will remove the "an" and restore the "a" because for uniform, that has
the verbal "y" constant sound as far as I know in English, and so
reading it out loud as either the word or the individual letters should
be correct.

Thank you for the confirmation.

**Review by angelikatyborska — APPROVED** [review]

**Review by ErikSchierboom — APPROVED** [review]

**Comment by kotp** [comment]
On 21/12/28 04:37AM, wolf99 wrote:
> Merged #1901 into main.

Thank you @wolf99 for merging!

**Pull request merged**